package.json

```diff
@@ -23,6 +23,8 @@
     "stack-pack-request": "2.3.1"
   },
   "devDependencies": {
-    "@gp-technical/stack-pack-standard": "^2.3.6"
+    "@gp-technical/stack-pack-standard": "^2.3.6",
+    "@gp-technical/eslint-config-stack": "^1.0.0",
+    "@gp-technical/stack-pack-app": "^2.0.2"
```

Comment:
stack-pack-gpapi shouldn't need stack-pack-app. The latter is for use in the app not the api.

Also the linting looks like it's not been added quite correctly. You shouldn't need the reference to standard-engine in package.json (line 20ish). See here for details: https://github.com/gp-technical/eslint-config-stack/blob/master/README.md.

Plus the way the linting is invoked is the old way as well. Again the details are in the Readme for eslint-config-stack as above.

I'd also try making stack-pack-util a peer dep (plus add it as a dev dep). That way we don't end up with multiple versions of the same package across the app. Only the app itself supplies the package with the other packages that need it pulling it in as a peer dependency.

Reply:
Thanks @ArthurSeaton for the comprehensive README.md .

I am applying the changes, please let me know if it's ok now.

Not fully understand about the stack-pack-util yet, can we discuss that later?